### DISCUSSION TITLE
No tag :latest for Docker images Beta Version

### DISCUSSION THREAD

**Discussion by corasaniti** [grafana/grafana#58818] [General]
Hi,

Why tag and push a Docker image with the :latest tag if it's still in beta?
The :latest tag should be left for the stable release
Thanks

**Comment by daniellee** [comment]
The latest tag is v9.2.5 and was pushed 5 days ago. I can check if we made a mistake during the release of the beta.

**Reply by daniellee** [reply]
Sorry if that caused you issues - it was a bug in our build pipeline that should be fixed now for future releases.

**Reply by corasaniti** [reply]
Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
